hawkbit-dmf/hawkbit-dmf-amqp/src/test/java/org/eclipse/hawkbit/amqp/AmqpMessageHandlerServiceTest.java

```diff
@@ -301,7 +301,7 @@ public void invalidEventTopic() {
         final MessageProperties messageProperties = createMessageProperties(MessageType.EVENT);
         final Message message = new Message(new byte[0], messageProperties);
         try {
-            amqpMessageHandlerService.onMessage(message, MessageType.EVENT.name(), TENANT, "vHost");
+            amqpMessageHandlerService.onMessage(message, "unknownMessageType", TENANT, "vHost");
```

Comment:
TODO: doublecheck

Reply:
has this been checked?

Reply:
@StefanKlt I think not. But I think this was maybe a copy-paste mistake, since the invocation with MessageType.EVENT should not produce any exceptions.
So, I think it should be like this. What do you guys think @schabdo ?

Reply:
The provided fix is correct. Indeed an unknown message type should be set. Current test was only executed successfully due to the missing event topic. So an `AmqpRejectAndDontRequeueException` is thrown anyway.